files/en-us/mozilla/add-ons/webextensions/manifest.json/version/index.html

```diff
@@ -28,7 +28,7 @@
  </tbody>
 </table>
 
-<p>Version of the extension, formatted as numbers and ASCII characters separated by dots. For the details of the version format, see the <a href="/en-US/docs/Toolkit_version_format">Version format</a> page. </p>
+<p>Version of the extension, formatted as numbers and ASCII characters separated by dots. For the details of the version format, see the <a href="https://developer.mozilla.org/en-US/docs/Mozilla/Toolkit_version_format">Version format</a> page. </p>
```

Comment:
```suggestion
<p>Version of the extension, formatted as numbers and ASCII characters separated by dots. For the details of the version format, see the <a href="/en-US/docs/Mozilla/Toolkit_version_format">Version format</a> page. </p>
```
no need for the full domain